db/schema.rb

```diff
@@ -20,8 +20,8 @@
   create_table "event_translations", force: :cascade do |t|
     t.integer  "event_id",     null: false
     t.string   "locale",       null: false
-    t.datetime "created_at",   null: false
-    t.datetime "updated_at",   null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
```

Comment:
@karimmtarek Can you please re-run you migrations from the beginning? Timestamps have a null constraint and it looks as if your schema was generated from an older version. Thanks!